app/main.py

```diff
@@ -1,13 +1,13 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
+    # write your code here
     pass
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
+    # write your code here
     pass
```

Comment:
The `format_single_linter_file` function is currently empty. You need to implement the logic for this function.